recipes/piawka/meta.yaml

```
@@ -0,0 +1,56 @@
+{% set name = "piawka" %}
+{% set version = "0.7.10" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  path: ../
+  url: https://github.com/novikovalab/piawka/archive/refs/tags/0.7.10.tar.gz
+  sha256: 511859b69525db7dc988ae55556e6fd31e2709f0e3a4de404f87abfadb97a273
+
+build:
+  noarch: generic
+  number: 0
+  run_exports:
+    - {{ pin_subpackage('piawka', max_pin="x") }}

```

```suggestion
    - {{ pin_subpackage('piawka', max_pin="x.x") }}

```
Recommended for "semantic versioning (0.x.x)"

---

revised accordingly